### ISSUE TITLE
Oculus Touch controllers disabled on startup (Oculus rift cv1)

### ISSUE THREAD

**Issue by vbeauregard** [Okabintaro/SubmersedVR#93]
Hi there,

When the game starts, the main screen show up without the touch controllers being responsive.  I can't pick anything on the menu, no lasers beam show up.  The only buttons that reacts are the system menu and home button.
The only in-game action that works is quitting the game.

I followed the guide, ran the install of Vortex (fresh, no QModManager to clear out), BepInEx for Subnautica and finally unzipped the content of the SubmersedVR file into my game folder.  I also added the -open

I have 2 log files for you:

[LogOutput1.log](https://github.com/Okabintaro/SubmersedVR/files/13681008/LogOutput1.log)
[LogOutput2.log](https://github.com/Okabintaro/SubmersedVR/files/13681009/LogOutput2.log)

LogOutput1.log: I started the game from the Steam Library after having set the advanced option "-vrmode openvr"
LogOutput1.log: 2nd try, still with the advanced option set to "-vrmode openvr" but this time I loaded up the SteamVR Home and started the game from there.

I noticed a warning msg in the logs:
**[Warning:SubmersedVR] Game was not started in VR. Don't load and apply SubmersedVR patches.**
That seems to indicate that the BepInEx and SubmersedVR install were successful but some config element prevents the game from being considered "started from VR"...

When I somehow managed to run the game in normal mode from whithin my VR headset (don't ask me how I pulled that one off!), I could use the controllers a bit and the fire was visible (which is nice) but I could not move using the thumbsticks and neither could I access the PDA so I figured that was not a good idea.

Keep me posted and thank you very much for your efforts.
I love the game and find it over-the-top immersive in VR so I'm not really interested in playing it in normal mode.
However, running the game without your mod means no save game at all which is not a practical solution.

**Comment by Okabintaro** [comment]
Yeah, the mod is still not getting loaded. Make sure you start the game in VR Mode. The logs don't give me more information unfortunately. It should do that when you launch from the dashboard so not sure why its not.
Also I never used Vortex tbh, just installed BepInEx and the mod like described in the README.

You should choose the second option when starting the game:

![image](https://github.com/Okabintaro/SubmersedVR/assets/103938900/690e090c-0e18-4b71-9749-c4ee54903413)

If you remembered it previously you can change it by clicking on the game and change the launch options there:

![image](https://github.com/Okabintaro/SubmersedVR/assets/103938900/29f57330-6a56-4b22-be46-2b841e138ef2)

**Comment by vbeauregard** [comment]
Thanks! 

I did both of these before reporting the issue but I'll give it another shot.
Another thing I noticed:   On my setup, the Oculus apps has to be up-and-running for SteamVR to detect the deadset.

I don't know if that would be a problem for the mod.

**Comment by vbeauregard** [comment]
Hey there again,

I found this article on the net in Unity forums.
[](https://forum.unity.com/threads/xr-settings-not-showing.948544/)

Is it possible that the latest Steam or Oculus drivers does not support the flag your mod use to evaluate if the HMD is up-and-running?
What's the new XR SDK plug-in system?  
Does it somewhat reflects that the old ways of handling XR have been made obsolete?
